modules/smart-agent_supervisor/detectors-supervisor.tf

```diff
@@ -36,7 +36,7 @@ resource "signalfx_detector" "process_state" {
   program_text = <<-EOF
     signal = data('supervisor.state', filter=${module.filtering.signalflow})${var.process_state_aggregation_function}${var.process_state_transformation_function}.publish('signal')
     detect(when(signal > ${var.process_state_threshold_critical})).publish('CRIT')
-    detect(when(signal < ${var.process_state_threshold_major})).publish('MAJOR')
+    detect(when(signal == 0 ).publish('MAJOR')
```

Comment:
I'd rather change default value of the `process_state_threshold_major` variable.

Reply:
But a threshold_major of 10 would involve only one possible code : 0 = stopped.
This let some possibility in the codebase to implement subsequent  return code < 10,  if one day superversor change his 10*code pattern design for return codes... 
Agreed. -> Pushed